index.js

```diff
@@ -154,6 +139,26 @@ function loadPackageMeta (pkgs, logos) {
   })
 }
 
+function requestOpts (pkg) {
+  pkg = decodeURIComponent(pkg)
+  var opts = {method: 'get', json: true}
+  var url = process.env.COUCH_URL_REMOTE || ExplicitInstalls.defaultRegistry
+  if (/^@/.test(pkg) && process.env.FRONT_DOOR_HOST) {
+    // fetch the module from our private npm On-Site
+    // instance (don't set proxy, and pass secret).
+    url = process.env.FRONT_DOOR_HOST
+    opts.qs = {
+      sharedFetchSecret: process.env.SHARED_FETCH_SECRET
+    }
+  } else if (process.env.PROXY_URL) {
+    // fetch from the global registry, we should set a
+    // proxy if it's provided.
+    opts.proxy = process.env.PROXY_URL
+  }
+  opts.url = url + '/' + pkg.replace('/', '%2f')
```

Comment:
The registry, as currently implemented, does not like the `@` symbol to be encoded as `%40`.

Silly bug, but figured I'd opt for this workaround for the time being -- would be a nice thing to patch upstream.